Moya-Gloss.podspec

```diff
@@ -26,8 +26,8 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |ss|
     ss.source_files  = "Source/*.swift"
-    ss.dependency "Moya", "8.0.0-beta.6"
-    ss.dependency "Gloss", "~> 1.0"
+    ss.dependency "Moya", "8.0.0"
+    ss.dependency "Gloss", "~> 1.1.1"
```

Comment:
not desired to restrict to a patch level release of Gloss, but that's just a nitpick since I can easily revert when doing changelog+release :)